CoreFoundation/URL.subproj/CFURLSessionInterface.c

```diff
@@ -31,6 +32,10 @@ static CFURLSessionMultiCode MakeMultiCode(CURLMcode value) {
     return (CFURLSessionMultiCode) { value };
 }
 
+CFStringRef CFURLSessionErrorDescription(int value) {
```

Comment:
`CFURLSessionCreateErrorDescription`

Reply:
New version with this change and your other suggestions.